clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -37,6 +37,7 @@
 #include "llvm/Object/IRObjectFile.h"
```

Comment:
hopefully we're planning on upstreaming this?

Reply:
Absolutely! I am told there are some refactorings coming in for ClangLinkerWrapper, so I wanted to get some pieces in for now so development of other parts, like the driver, can go ahead in parallel.